.github/workflows/build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
   upgrade:
     runs-on: ubuntu-latest
-    needs: ['lint']
+    needs: ['lint','upgrade-e2e']
```

Comment:
Should we wait for `unit-test` also?

Reply:
upgrade-e2e waits for unit-test so not required IMO.